src/views/admin/PersonDetailsView.vue

```diff
@@ -1494,17 +1576,28 @@
               v-if="personStore.currentPerson"
             >
               <div class="d-flex justify-sm-end">
-                <PersonDelete
-                  :disabled="isEditActive"
+                <PersonLock
```

Comment:
Kann man beim Laden vom Status ein: `<v-col v-else-if="personStore.loading"> <v-progress-circular indeterminate></v-progress-circular></v-col> `  hinzufügen? Sieht schöner aus und man vermeidet diesen aggressiven Switch.

Vielleicht auch beim 2FA Teil ;)

Reply:
Für den Sperren-Teil habe ich den Spinner eingebaut. 2FA wird in anderen Tickets bearbeitet, darum würde ich das hier erstmal außen vor lassen.